data/rest-data.go

```diff
@@ -16,14 +16,15 @@ import (
 )
 
 type HttpClient interface {
-    Do(req *http.Request) (*http.Response, error)
+	Do(req *http.Request) (*http.Response, error)
 }
 
 type RestData struct {
 	owner               string
 	repo                string
 	token               string
 	Config              *config.Config
+	WorkflowsEnabled    bool
```

Comment:
Whether or not the repository has GitHub Actions enabled is highly relevant to our evaluations as several of them review the contents of `.github/workflows` and use the GitHub REST API to review the configuration of workflows.

Once this change merges, we may also want to update the implementation of OSPS-BR-01.01 as it scans files in `.github/workflows` but if GHA is disabled for the repository it will incorrectly claim that CI input is sanitized because it reviews CI pipelines that are not in use.

Reply:
Good looking out.